pkg/kwok/controllers/node_controller.go

```diff
@@ -492,3 +492,12 @@ func (c *NodeController) Has(nodeName string) bool {
 func (c *NodeController) Size() int {
 	return c.nodesSets.Size()
 }
+
+// Get returns Has bool and corev1.Node if the node is existed
+func (c *NodeController) Get(nodeName string) (bool, corev1.Node) {
```

Comment:
```suggestion
func (c *NodeController) Get(nodeName string) (corev1.Node, bool) {
```

Usually, the bool that determines whether it exists is placed last

Reply:
Thank you. Please check this change.
https://github.com/kubernetes-sigs/kwok/compare/657f81e2535805cb4df0a6955505509e46ab8403..c7817a162e7eee66c48009316f12fb8b40e6dde1